### ISSUE TITLE
popularmoviesstage1.databinding folder not present?

### ISSUE THREAD

**Issue by jimandreas** [the4thfloor/PopularMovies#1]
Hello,   Thank you for sharing your kotlin project!   I have been poking around for kotlin projects to study.   As I have written "Popular Movies" in Java - I figure it would be interesting to find a kotlin version.

Here I am getting a build error on your most recent version where you added:

import udacity.nanodegree.popularmoviesstage1.databinding.ItemMovieBinding

But I don't seem to see the "databinding" portion of the tree present.

Thanks again for sharing your work!   Cheers, Jim

**Comment by the4thfloor** [comment]
@jimandreas the Kotlin version is a little bit outdate because Udacity can't review Kotlin thats why I started it again in Java :-(

I did not tried it in Android Studio but I guess the missing folder is the one which will be automatically created by the databinding processor

**Comment by jimandreas** [comment]
Yes you are right - I see an error in the databinding processing.   In browsing stackoverflow - it appears that databinding is still evolving rapidly.   I didn't find an obvious solution - and don't have time to play "mix and match" with all the versions.   Love your coding style - it is great!   Cheers, Jim

    Error:android.databinding.tool.util.LoggedErrorException: Found data binding errors.

**Comment by the4thfloor** [comment]
Okay I updated the used libraries... should work now

**Comment by jimandreas** [comment]
It is compiling and running great now.   Your code is really awesome and great for studying kotlin!   Thanks for the help on this, and thanks for sharing your work!    Your user interace graphics are really quite nice for this version of PopularMovies.   Cheers, Jim
